mlir/include/mlir/Dialect/XeGPU/IR/XeGPUOps.td

```diff
@@ -288,6 +301,8 @@ def XeGPU_PrefetchNdOp : XeGPU_Op<"prefetch_nd", []> {
   }];
 
   let arguments = (ins XeGPU_TensorDesc: $TensorDesc,
+                       Variadic<Index>: $offsets,
```

Comment:
Verifiers need to be extended to verify that provided offsets have correct rank

Reply:
added